html/canvas/tools/templates.yaml

```diff
@@ -108,271 +27,4 @@ w3c: |
-    href="http://wiki.whatwg.org/wiki/IRC">IRC</a>, etc) if you find any
-    problems.
-
-index.w3c: |
-    <!DOCTYPE html>
-    <title>Canvas tests - index</title>
-    <link rel="stylesheet" href="/html/canvas/resources/canvas-index.css">
-
-    <h1><code>&lt;canvas&gt;</code> tests</h1>
-
-index.category.item: |
-    <li><h3><a href="index.%s.html">%s.*</a></h3><p>%d test%s <a href="#" onclick="return expand(this)">expand</a></p>
-
-index.w3c.category.item: |
-    <li><h3><a href="index.%s.html">%s.*</a></h3><p>%d test%s</p>
-
-reportgen: |
-    <!DOCTYPE html>
-    <title>Canvas tests - report generator</title>
-    <link rel="stylesheet" href="../reportgen.css">
-    <script src="../reportgen.js"></script>
-    <p>This is mainly for my own use, so it is not designed to be user-friendly.
-    If anyone else wants to use it for some reason, just wait
-    until "tests not yet loaded" and "tests not yet completed" get down to zero, then click the
-    pass/fail button for any test it shows where it cannot work out the answer (or use the
-    <kbd>y</kbd>/<kbd>n</kbd> keys to choose for the magenta-highlighted case), then use the
-    buttons at the bottom to collect all the results.
-    <form id="f">
-    <p><label for="loading">Tests not yet loaded:</label> <input id="loading" value="0" readonly>
-    <p><label for="waiting">Tests not yet completed:</label> <input id="waiting" value="0" readonly>
-    <p><button type="button" onclick="showUnfinished()">Show uncompleted tests</button>
-       <button type="button" onclick="showAll()">Show all tests</button>
-       <button type="button" onclick="reloadAll()">Reload test cases serially</button>
-       <!--<button type="button" onclick="avoidFrameLimit()">HACK: work around frame limit</button>
-       <button type="button" onclick="makeObjects()">HACK: s/iframe/object/</button>-->
-    <hr>
-    <p><label for="passed">Detected passes:</label> <input id="passed" value="0" readonly>
-    <p><label for="failed">Detected fails:</label> <input id="failed" value="0" readonly>
-    <hr>
-    <table>
-    <tr>
-    <th>Test name
-    <th>Test case
-    <th>Pass?
-    <th>Fail?
-    <th>Notes
-    <script>
-    createTable( [
-    %(items)s
-    ] );
-    </script>
-    </table>
-    </form>
-    <hr>
-    <form method="post" action="../submitresults.cgi">
-    <button type="button" onclick="document.getElementById('report').value = genreport()">Generate test report</button><br>
-    <textarea name="report" id="report" cols="100" rows="10"></textarea><br>
-    <input type="submit" value="Submit results"> (Submissions will tend to be ignored unless there
-    is a good reason why they won't be, so don't use this form unless there is such a reason.)
-    </form>
-
-results: |
-    <!DOCTYPE html>
-    <title>Canvas tests - results</title>
-    <link rel="stylesheet" href="../results.css">
-    <table>
-    <col id="col1">
-    <tr>
-    <th>Test
+    %(scripts)s%(images)s
```

Comment:
Restore the newline to fix update-built-tests

Reply:
Thanks! I was having a heck of a time hunting that down.